frontend/packages/console-dynamic-plugin-sdk/src/shared-modules.ts

```diff
@@ -15,4 +35,26 @@ export const sharedPluginModules = [
   'react-redux',
   'redux',
   'redux-thunk',
-];
+] as const;
```

Comment:
Reference: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions

This changes the type of `sharedPluginModules` array from
```ts
string[]
```
to
```ts
readonly ["@openshift-console/dynamic-plugin-sdk" , ... , "redux-thunk"]
```